ReSharper.FSharp/src/FSharp.Psi.Features/src/Daemon/Highlightings/FSharpErrorUtil.fs

```diff
@@ -116,6 +116,10 @@ let getLambdaCanBeReplacedWarningText (replaceCandidate: IFSharpExpression) =
         sprintf "Lambda can be replaced with '%s'" x.QualifiedName
     | _ -> "Lambda can be simplified"
 
+let getIfCanBeReplacedWithConditionOperandWarningText needNegation =
+    if needNegation then "'if' expression can be replaced with condition operand negation"
+    else "'if' expression can be replaced with condition operand"
+
```

Comment:
Do we really need to add another function to this util? It should not be used for such simple cases, or it'll become too messy otherwise. Do we really need two different texts here?